{"uuid":"a545f558-df21-47fb-931f-78f981818297","name":"Scenario Outline: The user is presented with error message for invalid code in tryEditor for Max Consecutive Ones","children":["7dd54c4d-7eff-40ad-a5c5-1ba02bde07fd"],"befores":[],"afters":[{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"passed","stage":"finished","steps":[],"attachments":[{"name":"My screenshot","source":"6b0bde9d-be13-4313-96fe-23c0157ea2b0-attachment","type":"image/png"},{"name":"Myscreenshot","source":"0c769c6c-8bbd-4dbe-8a16-7bc87abb6b44-attachment"}],"parameters":[],"start":1696885483719,"stop":1696885483781},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885483782,"stop":1696885483782},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885483783,"stop":1696885483783},{"name":"Hooks.Hooks.a(io.cucumber.java.Scenario)","status":"skipped","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"AFTER_STEP is failed."},"stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1696885483784,"stop":1696885483784}],"start":1696885483715,"stop":1696885483784}